vendor.conf

```diff
@@ -16,7 +16,7 @@ github.com/imdario/mergo 50d4dbd4eb0e84778abe37cefef140271d96fade
 github.com/mattn/go-runewidth 14207d285c6c197daabb5c9793d63e7af9ab2d50
 github.com/mistifyio/go-zfs c0224de804d438efd11ea6e52ada8014537d6062
 github.com/mtrmac/gpgme b2432428689ca58c2b8e8dea9449d3295cf96fc9
-github.com/opencontainers/go-digest aa2ec055abd10d26d539eb630a92241b781ce4bc
+github.com/opencontainers/go-digest c9281466c8b2f606084ac71339773efd177436e7
```

Comment:
Is this really necessary for this PR? I can’t quickly see anything that makes a difference for concurrent execution in the changes.

Reply:
I usually try not to apply unnecessary changes. We need this one to get the hash of the digest (see https://github.com/opencontainers/go-digest/blob/c9281466c8b2f606084ac71339773efd177436e7/digest.go#L135).

Reply:
Ah, thanks. I missed that use of the new API.